chart/openfaas/templates/alertmanager-dep.yaml

```diff
@@ -22,10 +22,24 @@ spec:
         sidecar.istio.io/inject: "true"
         checksum/alertmanager-config: {{ include (print $.Template.BasePath "/alertmanager-cfg.yaml") . | sha256sum | quote  }}
     spec:
+      {{- if .Values.securityContext }}
+      securityContext:
+        fsGroup: 65534
```

Comment:
How did you pick `65534`? Can you link to documentation or a best-practices paper if you have one?

Reply:
That's the UID for the `nobody` user in most containers (the notable exception being the NATS one, which I believe is `FROM scratch`).

Reply:
@alexellis do you have any further concerns regarding this?

Reply:
https://github.com/openfaas/faas-netes/pull/509